WordPress/src/main/java/org/wordpress/android/ui/people/utils/PeopleUtilsWrapper.kt

```diff
@@ -0,0 +1,80 @@
+package org.wordpress.android.ui.people.utils
+
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.models.Person.PersonType
+import org.wordpress.android.ui.people.utils.PeopleUtils.FetchFollowersCallback
+import org.wordpress.android.ui.people.utils.PeopleUtils.FetchUsersCallback
+import org.wordpress.android.ui.people.utils.PeopleUtils.FetchViewersCallback
+import org.wordpress.android.ui.people.utils.PeopleUtils.InvitationsSendCallback
+import org.wordpress.android.ui.people.utils.PeopleUtils.RemovePersonCallback
+import org.wordpress.android.ui.people.utils.PeopleUtils.UpdateUserCallback
+import org.wordpress.android.ui.people.utils.PeopleUtils.ValidateUsernameCallback
+import javax.inject.Inject
+
+/**
+ * Injectable wrapper around PeopleUtils.
+ *
+ * PeopleUtils is consisted of static methods, which make the client code difficult to test/mock.
+ * Main purpose of this wrapper is to make testing easier.
+ */
+class PeopleUtilsWrapper @Inject constructor() {
+    fun fetchUsers(site: SiteModel, offset: Int, callback: FetchUsersCallback) =
+            PeopleUtils.fetchUsers(site, offset, callback)
+
+    fun fetchAuthors(site: SiteModel, callback: FetchUsersCallback) =
+            PeopleUtils.fetchAuthors(site, callback)
+
+    fun fetchRevisionAuthorsDetails(site: SiteModel, authors: List<String>, callback: FetchUsersCallback) =
+            PeopleUtils.fetchRevisionAuthorsDetails(site, authors, callback)
+
+    fun fetchFollowers(site: SiteModel, page: Int, callback: FetchFollowersCallback) =
+            PeopleUtils.fetchFollowers(site, page, callback)
+
+    fun fetchEmailFollowers(site: SiteModel, page: Int, callback: FetchFollowersCallback) =
+            PeopleUtils.fetchEmailFollowers(site, page, callback)
+
+    fun fetchViewers(site: SiteModel, offset: Int, callback: FetchViewersCallback) =
```

Comment:
For consistency, would it be better to format all these functions params like the ones below! wdyt?

Reply:
I agree. Done! db8361c5b71f0e55106ceeade812d73148d625fd